ocamltest/variables.ml

```diff
@@ -33,7 +33,7 @@ exception Variable_already_registered of string
 
 exception No_such_variable of string
 
-let default_exporter varname value = Printf.sprintf "%s=%s" varname value
+let default_exporter varname value = (varname, value)
```

Comment:
This is a belt and braces change - the code now relies on the environment representation `FOO=bar` so I didn't like that fact being exposed in custom exporters. This change means that a custom exporter can change both the name of the variable and the value (which is what it's actually used for) and then the formatting of it as `FOO=bar` is always done in `string_of_binding`.

Reply:
Strictly speaking, this isn't quite right. As far as the Unix kernel is concerned, the environment is simply a list of strings. The `FOO=bar` representation is only a user-level convention. If a program makes some other use of the environment, you might not be able to test it correctly. Of course, this is all rather theoretical.

Reply:
Quoting the Single Unix Specification:

> For a C-language program, an array of strings called the environment is made available when a process begins. [...]  These strings have the form name=value; names do not contain the character =. 

So, `VAR=value` may be "a user-level convention", but it's one that is set in stone in the standards.